.github/wordlist.txt

```diff
@@ -13,6 +13,7 @@ deserialization
 dotnet
 dotnetrocks
 durations
+enum
```

Comment:
If this was only used in the one place I suggested using the full word, then remove this.

```suggestion
```